$NetBSD: patch-aa,v 1.3 2013/10/03 11:12:43 obache Exp $

* disable DEBUG knob, it will be controlled with option.
* fixes manpath and installation command for unprivileged build.

--- Makefile.orig	2009-06-26 15:58:00.000000000 +0000
+++ Makefile
@@ -5,7 +5,7 @@
 
 VERSION= 1.7
 
-FDEBUG= 1
+#FDEBUG= 1
 
 CC?= cc
 YACC= yacc -d
@@ -76,12 +76,12 @@ regress:	fdm
 		cd regress && ${MAKE}
 
 install:	all
-		${INSTALLDIR} ${DESTDIR}${PREFIX}/bin
-		${INSTALLBIN} fdm ${DESTDIR}${PREFIX}/bin/
-		${INSTALLDIR} ${DESTDIR}${PREFIX}/man/man1
-		${INSTALLMAN} fdm.1 ${DESTDIR}${PREFIX}/man/man1/
-		${INSTALLDIR} ${DESTDIR}${PREFIX}/man/man5
-		${INSTALLMAN} fdm.conf.5 ${DESTDIR}${PREFIX}/man/man5/
+		${BSD_INSTALL_PROGRAM_DIR} ${DESTDIR}${PREFIX}/bin
+		${BSD_INSTALL_PROGRAM} fdm ${DESTDIR}${PREFIX}/bin/
+		${BSD_INSTALL_MAN_DIR} ${DESTDIR}${PREFIX}/${PKGMANDIR}/man1
+		${BSD_INSTALL_MAN} fdm.1 ${DESTDIR}${PREFIX}/${PKGMANDIR}/man1/
+		${BSD_INSTALL_MAN_DIR} ${DESTDIR}${PREFIX}/${PKGMANDIR}/man5
+		${BSD_INSTALL_MAN} fdm.conf.5 ${DESTDIR}${PREFIX}/${PKGMANDIR}/man5/
 
 uninstall:
 		rm -f ${DESTDIR}${PREFIX}/bin/fdm
